astropy/visualization/tests/test_units.py

```diff
@@ -40,6 +40,28 @@ def test_units():
         assert plt.gca().yaxis.get_units() == u.kg
 
 
+@pytest.mark.skipif(not HAS_PLT, reason="requires matplotlib.pyplot")
+def test_units_decorator():
+    @quantity_support()
```

Comment:
At the risk of bikesheding, I'm wondering if we could avoid requiring the empty pair of parens here. This does not seem possible within this approach (using `ContextDecorator`), so maybe that's a request I should report to CPython directly...

Reply:
+1 for an upstream feature request.